dependencies.txt

```diff
@@ -7,5 +7,5 @@ networkx
 owslib
 map2model
 hjson
-loopprojectfile
+loopprojectfile<=0.0.26
```

Comment:
@RoyThomsonMonash why?

Reply:
Actually in v3.1 the version number of loopprojectfile needs to be >=0.1.0.  Especially because that is what adds segNum and removes rank and type.

This is already set in v3.1 I'm not sure why it isn't here.